### DISCUSSION TITLE
FFUpdater struggles to update the apps 80% of the time

### DISCUSSION THREAD

**Discussion by n4tur4lb0rnch1ll4** [Tobi823/ffupdater#608] [Q&A]
Hello everyone,

FFUpdater will have a corrupt download 80% of the time (which is interesting because F-Droid or any other app doesn't face download issues at all) - does anyone have any idea what the issues is here?

Thank you very much in advance and best regards,

Flo

**Comment by Tobi823** [comment]
Which smartphone manufacturer are you using? Maybe Samsung or Huawai?

**Reply by n4tur4lb0rnch1ll4** [reply]
No, Google Pixel (7 Pro)

**Reply by Tobi823** [reply]
Can you make a screen recording? Maybe I can spot something odd.

**Reply by n4tur4lb0rnch1ll4** [reply]
Will do! Right after I wrote in here all downloads have been working so far...
